govuk-prototype-kit.config.json

```diff
@@ -7,7 +7,8 @@
   },
   "pluginDependencies": [{
     "packageName": "govuk-frontend",
-    "minVersion": "3.12.0"
+    "minVersion": "3.12.0",
+    "maxVersion": "4.7.0"
```

Comment:
TODO: update this if any further 4.x versions of `govuk-frontend` are released before v5. (unless it can be wildcarded?)